cloudbuild/presubmit.sh

```diff
@@ -37,16 +38,18 @@ cd /workspace
 case "$ACTION" in
   # Java code style check
   check)
-    ./mvnw spotless:check -P"${HIVE2_PROFILE}" && ./mvnw spotless:check -P"${HIVE3_PROFILE}"
+    $MVN spotless:check -P"${HIVE2_PROFILE}" && $MVN spotless:check -P"${HIVE3_PROFILE}"
     exit
     ;;
 
-  # Download maven and all the dependencies
+  # Build the Maven packages and dependencies
   build)
-    # Install all modules for Hive 2, including parent modules
-    $MVN install -DskipTests -P"${HIVE2_PROFILE}"
-    # Install the shaded deps for Hive 3 (all the other shaded & parent modules have already been installed with the previous command)
-    $MVN install -DskipTests -P"${HIVE3_PROFILE}" -pl shaded-deps-${HIVE3_PROFILE}
+    # Install shaded dependencies for Spark SQL
+    $MVN install -DskipTests -P sparksql -pl shaded-deps-sparksql
+    # Install all modules for Hive 2
```

Comment:
update readme for how to build hive-2 with sparksql support?

Reply:
I've simplified the process and removed the `sparksql-integration` profile altogether.